### PR TITLE
Added PT2 and PT3 filter types

### PR DESCRIPTION
**Pull request by Asizon**
This PR adds new filter type options to PidTuning/Filters Configurator tab, to go with a small part of https://github.com/betaflight/betaflight/pull/10727

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/pid_tuning.js
Comment (McGiverGim) [inline src/js/tabs/pid_tuning.js:1400]: I don't know if I like more this way or only with one push...
```suggestion
            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
                filterTypeValues.push("PT2", "PT3");
            }
```

**Review by McGiverGim — APPROVED** [review]
It's ok to me.

File: src/js/tabs/pid_tuning.js
Comment (Asizon) [reply src/js/tabs/pid_tuning.js:1400]: Interesting, but maybe i like the first way to best visual differentation betwen two filters type. Do you think that with a single push the performance will be better?

Comment (McGiverGim) [reply src/js/tabs/pid_tuning.js:1400]: I'm not too sure what I prefer, only was a comment 🙃

Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js:1400]: I prefer the former as it's easier to understand :upside_down_face:

**Review by McGiverGim — APPROVED** [review]

**Comment by haslinghuis** [comment]
I've to change `PTx` in 3 places. Can't this be done in one place?

**Comment by haslinghuis** [comment]
We need one test and the third approval here to test the new process. Milestone and RN tag is already set.

**Comment by blckmn** [comment]
AUTOMERGE: (fail)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (9/06/2021 22:49:35) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (None) -> fail
 - approver count at least three (3 approvers) -> pass

**Comment by haslinghuis** [comment]
@blckmn approver count passed? It's two at the moment :confused:

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:48:06) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (None) -> fail
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:48:14) -> pass
 - commit count less or equal to three (1 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> fail
 - assigned to an approver (None) -> fail
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed (10/06/2021 01:03:28) -> PASS
 - commit count less or equal to three (1 commits) -> PASS
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (3 approvers) -> PASS

**Comment by Asizon** [comment]
@haslinghuis I dont understund what are you refering with two places.

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
> 
> 
> @blckmn approver count passed? It's two at the moment 😕 The process should not run yet?

I am running it manually at the moment and doing testing :)

**Comment by blckmn** [comment]
Found the problem with the reviewers. If you submit two reviews, it adds you twice. Now counting the unique users who have approved the PR :)

**Review by blckmn — APPROVED** [review]

**Pull request merged**
